src/components/SearchBar/index.tsx

```diff
@@ -20,12 +20,15 @@ interface Props extends PopoverProps {
     dropdown?: string
     arrow?: string
     container?: string
+    dropdownList?: string
   }
   styles?: {
     dropdown?: CSSProperties
     arrow?: CSSProperties
     container?: CSSProperties
+    dropdownList?: CSSProperties
   }
+  renderItem?: (suggestion: string) => React.ReactNode
```

Comment:
hmm this pattern seems off to me 
Could we pass in just a React.ReactNode and pass the el as children for a more stylistically react way to decouple?

Reply:
It's a [common react pattern](https://jeffsegovia.dev/blogs/react-render-prop-pattern) but I agree it looks off because we don't use it. I replaced it with your suggestion: https://github.com/HyperPlay-Gaming/hyperplay-ui/pull/473/commits/32a7d55b1bd4534880244cacef431e305c394046